test/Microsoft.AspNetCore.Mvc.Core.Test/Filters/FilterCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Testing;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Filters
+{
+    public class FilterCollectionExtensionsTests
```

Comment:
I've just copied the tests from `FilterCollectionTests` and changed them to use the generic overload in stead. Does that make sense?